rust/signed_doc/src/signature/kid/key_version.rs

```diff
@@ -0,0 +1,19 @@
+//! COSE Signature Protected Header `kid` Role0 Key Version.
+
+use std::fmt::{Display, Formatter};
+
+/// Version of the Role0 Key.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct KeyVersion(u16);
```

Comment:
Let rename this to `KeyRotation` rather than `version` because it's the number of rotations of this role key that is used.